lib/howl/ui/editor.moon

```diff
@@ -367,16 +367,34 @@ class Editor extends PropertyObject
       return
 
     conf = @config_at_cursor
-    if conf.tab_indents and @current_context.prefix.is_blank
-      cur_line = @current_line
-      next_indent = cur_line.indentation + conf.indent
-      next_indent -= (next_indent % conf.indent)
-      cur_line.indentation = next_indent
-      @cursor.column = next_indent + 1
-    else if conf.use_tabs
+
+    if conf.use_tabs
       @view\insert '\t'
-    else
+      return
+
+    -- !NOTE!
```

Comment:
I'll try the settings out with master - since the post-content indents didn't behave as expected I wasn't sure which parts of the behaviour to consider correct.

_indent_: Fair enough, it is after all primarily a coders editor.